scripts/image-builder/configure.sh

```diff
@@ -18,6 +18,12 @@ sudo dnf install -y git osbuild-composer composer-cli ostree rpm-ostree \
     createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build \
     containernetworking-plugins expect
 
+RHEL93_REPOS=/usr/share/osbuild-composer/repositories/rhel-93.json
+if [[ ! -f "${RHEL93_REPOS}" ]]; then
+    sudo cp -v /usr/share/osbuild-composer/repositories/rhel-9.json "${RHEL93_REPOS}"
+    sudo sed -i "s,content/dist/rhel9,content/beta/rhel9,g" "${RHEL93_REPOS}"
+fi
```

Comment:
We may be forced to move to RHEL 9.3 before it's in GA and then we'd need to update this code.
Should we consider a full condition (i.e. beta + 9.3) ?

Reply:
Actually, we might want to stay on 9.2 EUS version and only update to next EUS (9.4), to keep our CI stable

Reply:
PTAL